tests/js/assets/js/yii.validation.test.js

```diff
@@ -0,0 +1,1598 @@
+describe('yii.validation', function () {
+    var VALIDATOR_SUCCESS_MESSAGE = 'should leave messages as is';
+    var VALIDATOR_ERROR_MESSAGE = 'should extend messages with according error(s)';
```

Comment:
I think my variation is more specific. With this message it's unclear what kind of result we actually expect from given data set.